### ISSUE TITLE
mods were changed in this recent update

### ISSUE THREAD

**Issue by outsideroom302** [jayvan/tempest-watch#8]
hi, i'm STC in-game and i've been using global 820 to get people to contribute to tempests

currently however some of the tempest prefix/suffixes aren't in game due to a recent patch
for example, consuming

http://poedb.tw/us/area.php?c=tempest  -  should have all the currently available datamined info
https://www.pathofexile.com/forum/view-forum/patch-notes

also i think with the rising popularity there may be an issue with griefing, is there any way that we could have it so only users with an account could participate?

regards

**Comment by outsideroom302** [comment]
Also if possible, an auto-refresh feature for the page would be greatly appreciated

**Comment by MMcCormick** [comment]
Both good suggestions, I think.

**Comment by jayvan** [comment]
I think that requiring an account to vote would be too much of an annoyance for many people to participate. I want to make it as easy as possible for someone to report tempests. Right now I've turned off all anti griefing except for limiting voting to once per IP per map per hour. If we find that griefing is a significant issue we can look into ways to combat it. I've updated some of the tempest descriptions, if there's any specific ones that you find are wrong feel free to open an issue or PR. I've also added a refresh button in commit 56bc8a8ed10b82f3c25d9b7ad27f9105cbae07f2.
